pages/submitted.tsx

```diff
@@ -14,9 +13,4 @@ const Submitted: NextPage = () => (
     </Layout>
 );
 
-export const getServerSideProps: GetServerSideProps = async ({res, query}) => {
-    await assertTokenIsValid(query, res);
-    return { props: {}};
-};
```

Comment:
same here - could you put this back?

Reply:
This code was exported from several places but couldn't see where its being used (though it looks like it is). I put it in api file and deleted the rest, thinking as its exported  anything that needs it can call it from there - is that not correct?

Reply:
Its used by Next - https://nextjs.org/docs/basic-features/data-fetching#getserversideprops-server-side-rendering

Its how you run code on the server side before rendering the page.
In this case we use it to prevent unauthorized access, so we definitely need it.

It must live on the page, otherwise Next won't find it.